docs/user/localization.rst

```diff
@@ -31,24 +31,30 @@ The language will be represented in the URL for your project.
 For example,
 a project that is in Spanish will have a default URL of ``/es/latest/`` instead of ``/en/latest/``.
 
-Projects with multiple translations (Sphinx-only)
--------------------------------------------------
+Projects with multiple translations
+-----------------------------------
 
 .. seealso::
 
    :doc:`guides/manage-translations-sphinx`
-     Describes the whole process for a documentation with multiples languages in the same repository and how to keep the translations updated on time.
+     Describes the whole process for a Sphinx documentation projects with multiples languages
+     in the same repository and how to keep the translations updated on time.
 
-This situation is a bit more complicated. Each language must have its own project. You will choose one to be the parent project, and add each of the other projects as "Translations" of the parent project.
+Each language must have its own project on Read the Docs.
+You will choose one to be the parent project,
+and add each of the other projects as "Translations" of the parent project.
 
-Let's use ``phpmyadmin`` as an example.
+Let's use ``godot`` as an example.
 
-The ``phpmyadmin`` project is the parent. Each translation then has its own project, for example ``phpmyadmin-spanish``. The *Language* for ``phpmyadmin-spanish`` is set to ``Spanish``. Then, on the  *Translations* page of the ``phpmyadmin`` project, ``phpmyadmin-spanish`` is set as translation.
+The ``godot`` project is the parent.
+Each translation then has its own project, for example ``godot-es``.
+The *Language* for ``godot-es`` is set to ``Spanish``.
+Then, on the  *Translations* page of the ``godot`` project, ``godot-es`` is set as translation.
 
 This has the results of serving:
 
-* ``phpmyadmin`` at ``https://phpmyadmin.readthedocs.io/en/latest/``
-* ``phpmyadmin-spanish`` at ``https://phpmyadmin.readthedocs.io/es/latest/``
+* ``godot`` at ``https://docs.godotengine.org/en/stable/``
+* ``godot-es`` at ``https://docs.godotengine.org/es/4.x/``
```

Comment:
The `stable` version just redirects to `4.x`, but as this is an illustration example, I'd use a matching `stable` branch here so that the only part of the path that is different is the language code.

```suggestion
* ``godot-es`` at ``https://docs.godotengine.org/es/stable/``
```